openedx/features/enterprise_support/tests/test_context.py

```diff
@@ -0,0 +1,44 @@
+"""
+Test the enterprise support APIs.
+"""
+from django.conf import settings
+from django.test.utils import override_settings
+
+from common.djangoapps.student.tests.factories import UserFactory, CourseEnrollmentFactory
+from openedx.core.djangolib.testing.utils import CacheIsolationTestCase, skip_unless_lms
+from openedx.features.enterprise_support.context import get_enterprise_event_context
+from openedx.features.enterprise_support.tests import FEATURES_WITH_ENTERPRISE_ENABLED
+from openedx.features.enterprise_support.tests.factories import (
+    EnterpriseCustomerUserFactory,
+    EnterpriseCourseEnrollmentFactory
+)
+from openedx.features.enterprise_support.tests.mixins.enterprise import EnterpriseServiceMockMixin
+
+
+@override_settings(FEATURES=FEATURES_WITH_ENTERPRISE_ENABLED)
+@skip_unless_lms
+class TestEnterpriseContext(EnterpriseServiceMockMixin, CacheIsolationTestCase):
+    """
+    Test enterprise support APIs.
```

Comment:
@rehanedly  need to change this to `Test enterprise event context APIs.`

Reply:
@ziafazal done